openqaoa/workflows/optimizer.py

```diff
@@ -409,14 +637,43 @@ def optimize(self, verbose=False):
         if self.compiled == False:
             raise ValueError('Please compile the QAOA before optimizing it!')
 
+        # timestamp for the start of the optimization
+        self.header['execution_time_start'] = int(datetime.utcnow().timestamp())
```

Comment:
confirm whether this is always correct (i.e. some issues found on google (https://stackoverflow.com/questions/28573873/python-datetime-utcnow-returning-incorrect-datetime)

Reply:
it was correct because timestamp is defined as 

> that have elapsed since 00:00:00 [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time) on 1 January 1970

but I have change the library so there is no confusion